packages/editor/src/private-apis.js

```diff
@@ -48,33 +34,19 @@ lock( privateApis, {
 	BackButton,
 	ExperimentalEditorProvider,
 	EntitiesSavedStatesExtensible,
-	GlobalStylesProvider,
 	EditorCanvas,
-	Header,
-	InserterSidebar,
-	ListViewSidebar,
+	EditorCanvasContentSlotFill,
+	GlobalStylesProvider,
 	mergeBaseAndUserConfigs,
-	PatternOverridesPanel,
 	PluginPostExcerpt,
-	PostActions,
-	PostPanelRow,
 	PreferencesModal,
 	usePostActions,
-	PostCardPanel,
-	PostStatus,
 	ToolsMoreMenuGroup,
 	ViewMoreMenuGroup,
-	PrivatePostExcerptPanel,
-	SavePublishPanels,
-	PostContentInformation,
-	PostLastEditedPanel,
 	ResizableEditor,
 	Sidebar,
-	TextEditor,
-	VisualEditor,
 
 	// This is a temporary private API while we're updating the site editor to use EditorProvider.
-	useAutoSwitchEditorSidebars,
```

Comment:
These changes here just clean up the non-used private editor APIs (since now we're using the higher level components)